internal/kafka/internal/services/data_plane_cluster.go

```diff
@@ -53,7 +53,7 @@ func (d *dataPlaneClusterService) GetDataPlaneClusterConfig(ctx context.Context,
 
 	dynamicCapacityInfo := map[string]api.DynamicCapacityInfo{}
 	if d.DataplaneClusterConfig.IsDataPlaneAutoScalingEnabled() {
-		dynamicCapacityInfo, _ = cluster.RetrieveDynamicCapacityInfo()
+		dynamicCapacityInfo = cluster.RetrieveDynamicCapacityInfo()
```

Comment:
Ok, but then there are 2 choices IMO:

1. Change the `RetrieveDynamicCapacityInfo()` so that no errors are returned
2. Explicitly ignore it by assigning the error to `_`

Reply:
Just saw not that you proposed to change `RetrieveDynamicCapacityInfo` in the linked comment. Ignore this.